### DISCUSSION TITLE
Regarding positioning and sizing of the Table of Contents (TOC)

### DISCUSSION THREAD

**Discussion by duvelix** [mmistakes/minimal-mistakes#4204] [Q&A]
Hello, I have some questions while using this theme to create a blog.

I've been having some difficulty with the TOC and I'm hoping for some guidance on how to adjust it.

<img width="820" alt="1" src="https://user-images.githubusercontent.com/35926730/220660787-3c8de2c5-19c4-4bb3-aaed-10c07f0f55d6.png">

1) Currently, as shown above, the width of the TOC is too small. So the title to exceed two lines. Is there a way to increase the width of the TOC?

<img width="999" alt="2" src="https://user-images.githubusercontent.com/35926730/220662148-ca84f748-787f-495f-b256-b6d01a0c9baf.png">

2) Increasing the width of the TOC seems to make the text space too tight. Can the location of the TOC be moved outside? (Like above)

I read the guide and didn't find anything about it. Would you be willing to provide any suggestions or resources that might be helpful?

**Comment by yueyejian13** [comment]
classes: wide

**Reply by duvelix** [reply]
If set to classes: wide, the TOC is fixed at the top. :(
